src/vs/platform/windows/common/windows.ts

```diff
@@ -239,6 +239,7 @@ export interface IWindowSettings {
 	menuBarVisibility: MenuBarVisibility;
 	newWindowDimensions: 'default' | 'inherit' | 'maximized' | 'fullscreen';
 	nativeTabs: boolean;
+	enableVibrancy: 'none' | 'light' | 'medium-light' | 'dark' | 'ultra-dark';
```

Comment:
Would you think `vibrancy` instead of `enableVibrancy` would be fit here to be better consistent with @sylveon's PR?

Reply:
Done!